### ISSUE TITLE
GenelandFancy or Geneland_fancy?

### ISSUE THREAD

**Issue by derele** [alicebalard/GenelandFancy#1]
Hi Alice,

it is a bit strainge that you have to 
`install_github("alicebalard/Geneland_fancy")`
and then
`library("GenelandFancy")`
the second is even undocumented (you see the right library name only from the installation success message).

I like the second more. Maybe rename your repository on github?

**Comment by alicebalard** [comment]
Yep, I did rename it "GenelandFancy" and am in the process of homogeneizing
all :)

*-----------------*

*Alice Balard*DVM, MSc
PhD student, Freie Universität, Berlin
workplace : +49 30 2093 6450
mobile : +49 176 4427 3225
alice.cam.balard@gmail.com
Ecology and Evolution of molecular Parasite-Host Interactions (HU/IZW)
https://www.biologie.hu-berlin.de/de/gruppenseiten/ecoevolpara

Institute of Biology, Humboldt University Berlin (HU)
Philippstr. 13, Haus 14, 10115 Berlin
and
Leibniz-Institut für Zoo- und Wildtierforschung (IZW)
Alfred-Kowalke-Straße 17, 10315 Berlin
http://www.leibniz-izw.de

2018-04-25 8:27 GMT+02:00 Emanuel Heitlinger <notifications@github.com>:
